Mathlib/AlgebraicGeometry/PrimeSpectrum/Noetherian.lean

```diff
@@ -107,4 +107,9 @@ instance : NoetherianSpace (PrimeSpectrum R) := by
   rw [isNoetherianRing_iff, isNoetherian_iff_wellFounded] at H
   exact (closedsEmbedding R).dual.wellFounded H
 
+lemma _root_.minimalPrimes.finiteOfIsNoetherianRing : (minimalPrimes R).Finite :=
+  minimalPrimes.bijection R
+    |>.set_finite_iff
+    |>.mpr NoetherianSpace.finite_irreducibleComponents
+
 end PrimeSpectrum
```

Comment:
Maybe belongs to a different file, say `RingTheory` somewhere, but then that file needs to import `AlgebraicGeometry`

Reply:
Probably we should put the prime spectrum folder under ring theory instead?